src/api_authentication_app_skeleton/spec/requests/api/log_ins/create_spec.cr

```diff
@@ -1,11 +1,11 @@
 require "../../../spec_helper"
 
-describe Api::SignIns::Create do
+describe Api::LogIns::Create do
```

Comment:
I guess this should be `Logins` since it is a noun? https://grammarist.com/spelling/log-in-login/ but I think operations should still be a verb so `LogInUser` is correct...I think..

Reply:
hmm.. yeah, I think `Logins::Create` would be more appropriate here.